bundle/Core/Search/Elasticsearch/QueryDSL/PrefixQuery.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\TagsBundle\Core\Search\Elasticsearch\QueryDSL;
+
+use Ibexa\Elasticsearch\ElasticSearch\QueryDSL\Query;
+
+final class PrefixQuery implements Query
+{
+    private ?string $field;
+
+    private ?string $value;
+
+    public function __construct(?string $field = null, ?string $value = null)
```

Comment:
Constructor property promotion here too.

Reply:
fixed in f4b8cce7ea0a8525641b58a5d8cd5841bd2a9ac7